src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/custom.py

```diff
@@ -1356,8 +1625,6 @@ def create_vmss(vmss_name, resource_group_name, image,
         master_template.add_resource(build_vmss_storage_account_pool_resource(
             'storageLoop', location, tags, storage_sku))
         vmss_dependencies.append('storageLoop')
-    else:
-        raise CLIError('Unsupported storage profile.')
```

Comment:
You might keep this, if for nothing else but a fallback (a new profile added that is unhandled)